cmd/derper/derper.go

```diff
@@ -303,7 +318,12 @@ func main() {
 		err = rateLimitedListenAndServeTLS(httpsrv)
 	} else {
 		log.Printf("derper: serving on %s", *addr)
-		err = httpsrv.ListenAndServe()
+		var l net.Listener
```

Comment:
`ln` instead of the ever-ambiguous `l` variable (looks like 1 or I or l or | in many fonts)